apps/sscs/sscs-bulk-scan/prod.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: helm.fluxcd.io/v1
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
 kind: HelmRelease
 metadata:
   name: sscs-bulk-scan
```

Comment:
If these have no parameters in them, do we need to include them?

Reply:
Not needed, but not easy to make the script aware of this and remove patches.Its just lift and shift of existing config, minimum needed for migration.